examples/installation/build.sh

```diff
@@ -31,29 +31,52 @@ EOF
 COMMAND="$1"
 
 prelaunch_simulator() {
```

Comment:
since this function is getting a bit complex, it would be worthwhile to keep in a shared script like with do for the Swift version